ts/events.ts

```diff
@@ -1,25 +1,24 @@
 import { createEventElement, createMultiDayEventElement, getFirstDayOfTheWeek, clearEvents, differenceBetweenTwoDatesInDays, dateIsInRange} from './utils.js';
 import { getEventsFromLocalStorage } from './services.js';
-
+import { Event } from './models.js';
```

Comment:
I have done it like u said, but I think its not the right way, it would be the right way for production. If we do it like this then we need tsconfig.json option ""allowImportingTsExtensions": true", which needs "noEmit: true" , that means we cant tsc our files. So i reverted the changes with the last commit to import from .js. Maybe im wrong, and there is a way.